.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ orbs:
 jobs:
   build-and-test:
     docker:
-      - image: circleci/node:12
+      - image: circleci/node:14
```

Comment:
Could we run the tests in both node 12 and 14? As 12 is the current LTS I think we should make sure the setup works for it.

Reply:
yep, I wanted to test it with `14`

Reply:
Yes I'm not suggesting necessarily dropping 14 but rather also testing 12 so that CI can help catch issues we might not have in our local environment.

Reply:
Yeah, we'll do that definitely! I'll refactor the configuration in the next PR. Makes sense?

Reply:
Cool!

Reply:
I have to examine Jest vm/env in more depth.